src/commentscan.l

```diff
@@ -1338,15 +1338,16 @@ STopt  [^\n@\\]*
 
   /* --------- handle arguments of page/mainpage command ------------------- */
 
-<PageDocArg1>[^\n]*"\\ilinebr @ianchor"\{[^\]\n]*\}{B}{FILE} { // special case where the Markdown processor has rewritten
+<PageDocArg1>[^\n]*"\\ilinebr @ianchor/"\{[^\]\n]*\}{B}{FILE} { // special case where the Markdown processor has rewritten
```

Comment:
I see a `/` appear in `@ianchor` so I do not expect this rule will ever match, probably breaking the fix for issue #9254. Did you test this?

Reply:
I tested it on the current problem.
I just now tested it on the problem from #9254 and see that there is a problem again  e.g. in pages.html

Correct after fix for 9254:

![image](https://user-images.githubusercontent.com/5223533/227006405-92bc523e-4132-45e6-9fe9-66ea52d4c76d.png)


After my "fix":
![image](https://user-images.githubusercontent.com/5223533/227006618-dcc177c9-6193-4f21-8102-ed55fc9f06f4.png)

So, unfortunately, I have to dive into it again ...

Reply:
@albert-github As I tried to explain the rule at line 1341 above needs to match the rewritten `@page` command in Markdown to fix issue #9254, and because if the literal `/` that is introduced in the rule it no longer matches...

You probably meant to write this instead (I moved the `/` outside the `""`):

```.l
<PageDocArg1>[^\n]*"\\ilinebr @ianchor"/\{[^\]\n]*\}{B}{FILE} {
```